components/Nav/SearchSidebar/SearchSidebar.jsx

```diff
@@ -25,6 +25,7 @@ import { useRouter } from 'next/router'
 import useUserStore from 'store/use-user-state'
 import { withAlgorandPriceQuery } from '@/hooks'
 import useSearchFilter from '@/hooks/useSearchFilter'
+import { filter } from 'lodash'
```

Comment:
Can you delete this import since it is not used?

Reply:
Okay